system/db/schema/mysql/20190826085348.migrate-gplus-google.up.sql

```diff
@@ -0,0 +1,5 @@
+/* migrates existing credentials */
+UPDATE sys_credentials SET kind = 'google' WHERE kind = 'gplus';
+
+/* migrates existing settings. */
+UPDATE sys_settings SET name = REPLACE(name, '.gplus.', '.google.');
```

Comment:
Add `WHERE name LIKE 'auth.external.providers.gplus.%'`, just to be sure we're not breaking anything else.